src/App.tsx

```diff
@@ -1,27 +1,78 @@
 import React from 'react';
 import './App.scss';
+import { Clock } from './Components/Clock/Clock';
 
-const App: React.FC = () => {
-  const timerId: NodeJS.Timer = setInterval(() => {
-    const date: Date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  // eslint-disable-next-line
-  console.log(timerId);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type State = {
+  showClock: boolean,
+  clockName: string,
 };
 
+class App extends React.Component<{}, State> {
+  state = {
+    showClock: true,
+    clockName: 'React clock',
```

Comment:
I'd move this functionality to clock - logically it shouldn't be state of app.
but maybe i'm wrong